released/packages/coq-wasm/coq-wasm.0.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+version: "0.1"
+synopsis: "Wasm formalisation in Coq"
+description:
+  "Wasm formalisation in Coq, following the AFP formalisation of Conrad Watt"
+maintainer: ["Xiaojia Rao" "Martin Bodin"]
+authors: [
+  "Martin Bodin" "Philippa Gardner" "Jean Pichon" "Xiaojia Rao" "Conrad Watt"
+]
+license: "MIT"
+homepage: "https://github.com/WasmCert/WasmCert-Coq"
+bug-reports: "https://github.com/WasmCert/WasmCert-Coq/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "coq" {>= "8.16" & <= "8.17.1"}
+  "coq-compcert" {>= "3.11"}
+  "coq-mathcomp-ssreflect" {>= "1.16.0" & <= "1.17.0"}
+  "coq-itree" {>= "3.1.0"}
+  "coq-parseque" {>= "0.1.0"}
+  "cmdliner" {>= "1.1.0"}
+  "linenoise" {>= "1.4.0"}
+  "mdx" {>= "1.9.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+install: [
+[make "install"]
+]
```

Comment:
```suggestion
```
The `Makefile` just has a single action `default` that calls `opam` again. That won't work, let's have Dune handle the install...

Reply:
I see -- this was my first PR on the coq repo and I was blindly looking at some previous PRs to submit this one. Thanks a lot for the corrections!